plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1071,7 +1075,7 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
             -- For versions other (i.e. larger) than V1, the first input must be in range [0.255].
             -- See Note [How to add a built-in function: simple cases]
             _ -> makeBuiltinMeaning
-              (\(n :: Word8) xs -> BS.cons n xs)
+              BS.cons
```

Comment:
How about we settle on explicitly providing a Haskell type signature for every single builtin? It's something that's been on mind for a while, I think it would generally make things clearer.

Reply:
> Yes, nice idea but perhaps in another PR? 

Yep.

> For now could you rely on TypeApplications?

Will do.

Reply:
Love it.